lib/utils/remote-build.ts

```diff
@@ -405,8 +405,10 @@ async function getRemoteBuildStream(
 	// We only show the spinner when outputting to a tty
 	if (process.stdout.isTTY) {
 		const visuals = getVisuals();
+		const uploadUrl =
+			process.env.BALENARC_BUILDER_URL || `https://builder.${build.baseUrl}`;
 		uploadSpinner = new visuals.Spinner(
-			'Uploading source package to balenaCloud',
+			`Uploading source package to ${uploadUrl}`,
```

Comment:
This results in a log statement that looks for example like:
```
Uploading source package to https://builder.1ae7c2471111a3b24f64da7f8286014f.bob.local
```
when `BALENARC_BALENA_URL` is exported with value _1ae7c2471111a3b24f64da7f8286014f.bob.local_